messages/ar.json

```diff
@@ -0,0 +1,38 @@
+{
+	"@metadata": {
+		"authors": [
+			"MusikAnimal"
+			, "Wahrani"
+			, "FShbib"
+			, "Michel Bakni"
+		]
+	},
+	"assessment": "الجودة",
+	"config-subpage": "إعدادات الصفحات الأكثر مشاهدة",
+	"daily-average": "المتوسط اليومي",
+	"date-range": "$1 إلى $2",
+	"importance": "الأهمية",
+	"index-page-dont-edit": "رجاءً لا تعدل هذه الصفحة",
+	"index-page-header-1": "الجدول أدناه تمثيلٌ لجدول [[نص ويكي|بنص الويكي]] للإعدادات المستخدمة في توليد الصفحات الأكثر مشاهدة لمشاريع الويكي.",
+	"index-page-header-2": "يمكن إيجاد الإعدادات الفعليّة في $1.",
+	"index-page-overriden": "ستُزال جميع التغييرات على هذه الصفحة خلال التحديث القادم.",
+	"index-page-help": "لم يُحدّث مشروعك؟ فضلًا راجع $1.",
+	"index-page-help-link": "نصائح لإصلاح المشاكل ",
+	"limit": "الحد",
+	"list": "القائمة",
+	"list-of-projects": "قائمة المشاريع",
+	"page-title": "العنوان",
+	"period": "الفترة",
+	"popular-pages": "الأكثر مشاهدة",
+	"rank": "المرتبة",
+	"report": "التقرير",
+	"report-bug": "للإبلاغ عن أخطاء، فضلًا راسلنا في [[meta:User talk:Community Tech bot|صفحة نقاش  بوت المجتمع]] على الميتا.",
+	"report-comment": "التغييرات أدنى هذا السطر ستُزال في التحديث القادم.",
+	"report-header": "هذه قائمة الصفحات ضمن مواضيع $1 إلى جانب مشاهداتها، بما في ذلك جميع التحويلات.",
+	"total-views": "إجمالي المشاهدات",
+	"updated": "تاريخ التحديث",
+	"views": "المشاهدات",
+	"wikiproject": "مشروع ويكي",
+	"edit-summary" : "تحديث تقرير الصفحات الأكثر مشاهدة" ,
+        "unknown":"غير معروفة"
```

Comment:
should be indented with one tab, not 8 spaces

Reply:
done